semantic/src/Serializing/Format.hs

```diff
@@ -16,19 +19,22 @@ import Data.ProtoLens.Encoding as Proto
 import Data.ProtoLens.Message (Message)
 import Language.Haskell.HsColour
 import Language.Haskell.HsColour.Colourise
+import Source.Loc
 import Text.Show.Pretty
 
 data Format input where
-  DOT         :: (Ord vertex, ToGraph graph, ToVertex graph ~ vertex) => Style vertex Builder -> Format graph
-  JSON        :: ToJSON input                                         =>                         Format input
-  Show        :: Show input                                           =>                         Format input
-  Proto       :: Message input                                        =>                         Format input
```

Comment:
This looks absurd and would have generated a gory diff anyway, so I hit it with `ormolu`.